contrib/ruby/ext/trilogy-ruby/cext.c

```diff
@@ -1135,6 +1148,7 @@ RUBY_FUNC_EXPORTED void Init_cext()
     id_ivar_rows = rb_intern("@rows");
     id_ivar_query_time = rb_intern("@query_time");
     id_connection_options = rb_intern("@connection_options");
+    id_max_allowed_packet = rb_intern("@max_allowed_packet");
```

Comment:
I wasn't sure if this should live in its own instance variable or in the `@connection_options`. It

Reply:
Ah, now I follow the question you asked me before. If you are passing this in to `Trilogy.new` then I think it's in `@connection_options` already, so I'd prefer to use it there and avoid the extra instance variable.

But I guess it's a bit awkward because unlike the other connection_options we're not actually using this value to connect 🤔. Does that make this a `query_flag` instead?